var/spack/repos/builtin/packages/libtiff/package.py

```diff
@@ -40,6 +40,7 @@ class Libtiff(CMakePackage, AutotoolsPackage):
 
     license("libtiff")
 
+    version("4.7.0", sha256="67160e3457365ab96c5b3286a0903aa6e78bdc44c4bc737d2e486bcecb6ba976")
```

Comment:
#46999 adds 4.7.0 as well:
```suggestion
    version("4.7.0", sha256="67160e3457365ab96c5b3286a0903aa6e78bdc44c4bc737d2e486bcecb6ba976")
```
The GitLab CI pipelines of #46999 is currently running.

After it is merged, we can commit the suggestion as 4.7.0 is then existing upstream,
and rebase the PR to the merged commit of #46999 on develop.

I'd then approve and enable auto-merge.

If all goes well, is would be possible to merge it this week.